pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -148,18 +148,18 @@ func createTargetConfig(c TargetConfigController, recorder events.Recorder, oper
 	return false, nil
 }
 
-func loglevelToKlog(logLevel operatorv1.LogLevel) string {
+func loglevelToZap(logLevel operatorv1.LogLevel) string {
 	switch logLevel {
 	case operatorv1.Normal:
```

Comment:
As mentioned in the description, having these be 1:1 match spec to the etcd log level would make it easier I believe.

Reply:
I agree but `operatorv1.LogLevel` is an enum[1] for v1 API I think the map makes sense here probably. Most of these changes will be from `info` to `debug`.

[1] https://github.com/openshift/api/blob/5935a5beec4bb8e1e81dd0fe9ebc2af36b9a09ae/operator/v1/types.go#L89

Reply:
Ack! So the log-level conversion done here looks good to you?

Reply:
well logically this can be simplified a bit right but otherwise lgtm.

```
case operatorv1.Debug, operatorv1.Trace, operatorv1.TraceAll:
 return "debug"
default:
  return "info"
```

Reply:
yes 💯  of course, left it  this way to easily demonstrate the question I had. :)